code/src/cljs/sixsq/nuvla/ui/notifications/views.cljs

```diff
@@ -409,13 +420,138 @@
                   :style      {:padding-bottom 8}} "Condition"]
    [ui/TableCell
     [ui/Dropdown {:selection true
+                  :disabled (boolean (metric-name->default-condition metric-name))
                   :options   (vec
                                ((keyword metric-name)
                                 (get criteria-condition-options collection)))
                   :error     (and validate-form? (not (seq condition)))
+                  :value     condition
                   :on-change (ui-callback/value
                                #(on-change :criteria {:condition %}))}]]])
 
+(def metric-name->use-other-translation-key
+  {"disk" [:subs-notif-disk-use-other]
+   "network-rx" [:subs-notif-network-use-other]
+   "network-tx" [:subs-notif-network-use-other]})
+
+(def metrics-with-customizable-dev-name #{"network-rx" "network-tx" "disk"})
```

Comment:
Should those values be stored in vars, i.e. `(def network-rx "network-rx")` etc for usage everywhere?

Reply:
And a follow-up question: If I also need these config options inside the `notifications.utils` namespace, where would I put them? Inside the utils namespace or in a new namespace `notifications.config`?

Reply:
Put them in utils ns